HHS/Head-Start-TTADP
# [TTAHUB-1841] Some feedback from session & training report demos

## Description of change

- Design asked to adjust margin between cards on training reports and goal dashboard
- Design asked to adjust margin between label and value on the "View" training reports
- Design asked to move the "incomplete form/sessions" message to **above** the submit button group
- Automatically set a training report to "in progress" if you start editing it
- Simplified the success message on the landing page since it was broken (we can iterate on this, but trying to put it in an MVP state)
-  When adding a session, the form should load at the top of the page rather than the scroll position of the previous view.
- Add Back to training reports link at the top of form pages
- When creating a session, the "complete session" section should have a state of "not started" rather than "in progress" before the section has been entered.
- The H2 on the Complete sessions section should say "Complete session" rather than "complete event"
- Update content of success alert when submitting session to "You successfully submitted the session.“
- Update responsive behavior on the Event Cards
- Move "save draft" alert above the buttons (session, event, and AR forms)

## How to test
 
Confirm that the issues brought up by the designers were addressed.
As a sanity check, run through AR, Event, and session creation to make sure that nothing got tweaked (stuff was changed all the places)

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1841

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
